release/gcb/docker_tag_push_lib.sh

```diff
@@ -54,24 +54,22 @@ function add_extra_artifacts_to_tar_images() {
   fi
 
   for TAR_PATH in "${OUT_PATH}"/docker/*.tar.gz; do
-    BASE_NAME=$(basename "$TAR_PATH")
-    TAR_NAME="${BASE_NAME%.*}"
-    IMAGE_NAME="${TAR_NAME%.*}"
-
     # if no docker/ directory or directory has no tar files
-    if [[ "${IMAGE_NAME}" == "*" ]]; then
+    if [[ "${TAR_PATH}" == "${OUT_PATH}/docker/*.tar.gz" ]]; then
```

Comment:
For my own knowledge, how does this work? In my limited bash knowledge it looks like we get all files matching `"${OUT_PATH}/docker/*.tar.gz"` then check if it matches `"${OUT_PATH}/docker/*.tar.gz"` which is always true?

I have the same question about the old code so I think its correct I just don't understand it so was curious

Reply:
The quotes around `"${OUT_PATH}/docker/*.tar.gz"` avoid the expansion of the wildcards.  Therefore, this condition is only true if there are no files matching the pattern 2 lines above.